pkg/graph/resolvers/model_plan_phase_test.go

```diff
@@ -0,0 +1,179 @@
+package resolvers
+
+import (
+	"time"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/cmsgov/mint-app/pkg/storage"
+	"github.com/cmsgov/mint-app/pkg/userhelpers"
+
+	"github.com/cmsgov/mint-app/pkg/email"
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/shared/oddmail"
+)
+
+func (suite *ResolverSuite) TestSendEmailForPhaseSuggestionByModelPlanID() {
+	mockController := gomock.NewController(suite.T())
+	mockEmailService := oddmail.NewMockEmailService(mockController)
+	mockEmailTemplateService := email.NewMockTemplateService(mockController)
+
+	planName := "Plan For Milestones"
+	plan := suite.createModelPlan(planName)
+
+	collaboratorInput := &model.PlanCollaboratorCreateInput{
+		ModelPlanID: plan.ID,
+		UserName:    "CLAB",
+		TeamRoles:   []models.TeamRole{models.TeamRoleLeadership},
```

Comment:
The AC for the user story says it should go to `model_leads`, not to `leadership`, are you trying to assert this user shouldn't get an email?

Reply:
the model lead here would be whoever created the model. (maybe some comments would be helpful)